server/routes/user.routes.js

```diff
@@ -9,14 +9,18 @@ router.get("/loginReactPageHere", (req, res) => {
   res.json({ message: "Invalid email or password, try again" });
 });
 router.get("/:id", UserController.getUserByUserId);
-router.post("/:id/languages", UserController.updateUser);
+router.patch("/:id", UserController.updateUserById);
+router.patch("/", UserController.updateUserByEmail);
 router.post("/register", UserController.registerUser);
 router.post(
   "/login",
   passport.authenticate("local", {
     failureRedirect: "/user/loginReactPageHere",
   }),
-  UserController.logInUser
+  UserController.logInUser,
 );
+router.post("/otpEmail", UserController.sendOtpEmail);
+router.post("/formEmail", UserController.sendFormLinkEmail);
+router.post("/verifyOTP", UserController.verifyOTP);
```

Comment:
These don't seem restful. Can we change them to as follows:

```suggestion
router.post("/otp/emails", UserController.sendOtpEmail);
router.post("/forms/emails", UserController.sendFormLinkEmail);
router.post("/otp/verification", UserController.verifyOTP);
```

Reply:
Changed! Thanks for the heads up!